Resources/public/css/styles.css

```diff
@@ -345,6 +345,28 @@ td.sonata-ba-list-field .editable-empty:not(.editable-open) {
     width: auto; /* See https://github.com/sonata-project/SonataAdminBundle/issues/2950 */
 }
 
+/* icheck */
+.checkbox div[class^="icheckbox"],
```

Comment:
In this case I don't think this is a sonata issue but an ICheck / AdminLTE issue.

cc @sonata-project/contributors